crates/proof-of-sql/src/base/arrow/mod.rs

```diff
@@ -0,0 +1,46 @@
+//! This module provides conversions and utilities for working with Arrow data structures.
+
+#[cfg(feature = "arrow")]
```

Comment:
You probably want `#[cfg(feature = "arrow")]` on `pub mod arrow` in `base/mod.rs` instead and remove all of them here

Reply:
ok